runtime/common/src/zenlink.rs

```diff
@@ -0,0 +1,267 @@
+use spacewalk_primitives::{Asset, CurrencyId};
+use crate::stellar::{USDC_ISSUER, TZS_ISSUER, BRL_ISSUER};
+
+use zenlink_protocol::{LOCAL, NATIVE};
+pub type ZenlinkAssetId = zenlink_protocol::AssetId;
+
+fn discriminant(currency: &CurrencyId) -> u8 {
+	match currency {
+		CurrencyId::Native => 0,
+		CurrencyId::XCM(_) => 1,
+		CurrencyId::Stellar(_) => 2,
+		CurrencyId::ZenlinkLPToken(_, _, _, _) => 6,
```

Comment:
Nit: 
Just something that came to my mind here, we don't actually have to skip 3, 4 and 5, do we? Maybe it's best if we set the discriminant of the LP token to `3`? Or is there any downside to this @adelarja?

Reply:
We need to skip those values because the zenlink protocol uses [this](https://github.com/zenlinkpro/Zenlink-DEX-Module/blob/7662acbff71cea3b35db918ceb9370e8929be991/zenlink-protocol/src/primitives.rs#L65C2-L71C3) function to generate the LP asset id, and the LP token discriminant is hardcoded to 6.

Reply:
Ahhhhh right. Thanks for pointing that out. We should definitely mention that here in a comment though.

Maybe it would be even better if we don't use Zenlink's default implementation of `GenerateLpAssetId` but implement this locally ourselves now? Then we could change it to use the discriminant of `3`. Plus, we wouldn't need to rely on Zenlink to never change that function implementation again. And it would make it a little more obvious how we generate the LP token asset ID. WDYT @adelarja?

Reply:
Makes sense. I'd rename `SpacewalkLPGenerate` to `ZenlinkLPGenerate` since this is unrelated to spacewalk. Looks good otherwise 👍

Reply:
@ebma I've made the changes you suggested to me. I have also added the Zenlink Protocol configuration to the Amplitude runtime.